ct-app/core/model/peer.py

```diff
@@ -14,6 +18,12 @@ def __init__(self, id: str, address: str):
         self.economic_model = None
         self.reward_probability = None
 
+    def version_is_old(self, min_version: str or Version) -> bool:
+        if isinstance(min_version, str):
+            min_version = parse_version(min_version)
+
+        return self.version < min_version
```

Comment:
Should this not be 
```suggestion
        return self.version <= min_version
```

Reply:
Tests for this functionality would be nice with different valid and invalid versions.

Reply:
The method check if the version is too old. It's too old only if it's version is smaller than the minimum requirement. If it's equal to the minimum requirement, the node version is acceptable.